src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -6315,7 +6315,10 @@ def test_create_image_version_with_region_cvm_encryption_pmk(self, resource_grou
 
         self.cmd('sig image-version delete -g {rg} --gallery-name {gallery} --gallery-image-definition {image} --gallery-image-version {version}')
         time.sleep(60)  # service end latency
+        self.cmd('snapshot delete -g {rg} -n {snapshot1}')
+        self.cmd('disk delete -g {rg} -n {disk1} -y')
         self.cmd('sig image-definition delete -g {rg} --gallery-name {gallery} --gallery-image-definition {image}')
+        time.sleep(60)
```

Comment:
Why do we need to sleep for 60 seconds during the test?

Reply:
Without this, the test will report an error.
Error message:
E           azure.core.exceptions.ResourceExistsError: (CannotDeleteResource) Cannot delete resource while nested resources exist. Some existing nested resource IDs include: 'Microsoft.Compute/galleries/gallery_iaxyzqwjxotm/images/image1'. Please delete all nested resources before deleting this resource.
E           Code: CannotDeleteResource
E           Message: Cannot delete resource while nested resources exist. Some existing nested resource IDs include: 'Microsoft.Compute/galleries/gallery_iaxyzqwjxotm/images/image1'. Please delete all nested resources before deleting this resource.